### PR TITLE
New Version Implementation

### PR DESCRIPTION
**Pull request by salil-naik**
- Changed the variables
- Created typeface-style classes
- Added a temporary main.css and index.html to test the classes

screencapture-127-0-0-1-5500-src-index-html-2021-06-11-12_53_51

### CODE REVIEW

**Comment by salil-naik** [comment]
- Added light color variables and created classes
- Updated preview section to test the classes

image

**Review by ankitboghra — CHANGES_REQUESTED** [review]
Please remove the `package-lock.json`
We do not want to change the `lockFileVersion` from 1 to 2

**Comment by salil-naik** [comment]
- [f68251f](https://github.com/maticnetwork/matic-design-system/pull/45/commits/f68251f93ad4b149a1ef8c9011713919866eb002) Added the dark style color variables. 

- [933c53e](https://github.com/maticnetwork/matic-design-system/pull/45/commits/933c53e7dada5a32e272954f0bbf0fadbce74a71) Added Luminance function that checks for the luminance (brightness) of the bg-colour and returns luminance of the colour (a value ranging from 0-1). Further, this value is used to give either white or black colour to the text. (shown in the picture below) **I'll improve it even more as I learn more about it.** 

screencapture-127-0-0-1-5500-src-index-html-2021-06-14-20_24_38 1

**Comment by salil-naik** [comment]
> Please remove the `package-lock.json`
> We do not want to change the `lockFileVersion` from 1 to 2

Done.

**Comment by salil-naik** [comment]
- [eec484d](https://github.com/maticnetwork/matic-design-system/pull/45/commits/eec484dd39d214c07e96e8d54a489384d886a7b6) Added classes for corner, shadows and spacing. 
 
 
shadow-corner-spacing

**Comment by salil-naik** [comment]
Added button classes. 
Took into consideration various sizes, colour variants, states like hover pressed and disabled. 

image

**Comment by salil-naik** [comment]
A demo file using all the column classes. Gutter space and container size can be changed by the developer by overwriting the CSS variables.

localhost_5500_src_demo_grid html (1)

File: src/demo/css/style.css
Comment (ankitboghra) [inline src/demo/css/style.css:1880]: not sure about this, let us discuss this

Comment (salil-naik) [reply src/demo/css/style.css:1880]: This is not part of the design system. It is used for the demo purpose.

File: src/demo/css/style.css
Comment (salil-naik) [reply src/demo/css/style.css:1766]: This is not part of the design system. It is used for the demo purpose.

File: src/demo/css/style.css
Comment (salil-naik) [reply src/demo/css/style.css:1565]: Noted.

File: src/demo/css/style.css
Comment (salil-naik) [reply src/demo/css/style.css:1349]: Yes, I fixed this in another branch. Will merge that here once approved.

File: src/demo/css/style.css
Comment (salil-naik) [reply src/demo/css/style.css:272]: Usually, only 1 CSS library is used in the project, right?

File: src/demo/css/style.css
Comment (salil-naik) [reply src/demo/css/style.css:67]: Noted.

**Review by nitinmittal23 — APPROVED** [review]

**Review by ankitboghra — APPROVED** [review]

**Pull request merged**
